src/unix/core.c

```diff
@@ -303,6 +303,9 @@ int uv_loop_alive(const uv_loop_t* loop) {
     return uv__loop_alive(loop);
 }
 
+int uv_pending(uv_loop_t* loop) {
+    return (QUEUE_EMPTY(&loop->pending_queue) == 0);
+}
```

Comment:
Style issues:

1. Please put two blank lines between functions.

2. `loop` can be const.

3. The indent is off.

4. The parentheses are not necessary.

5. For consistency with the code a few lines up, it would be better to write it as `!QUEUE_EMPTY(&loop->pending_queue)`.